styles/Checkboxes.module.css

```diff
@@ -0,0 +1,21 @@
+.container {
+  font-size: 25px;
+  width: 100%;
+}
+
+.container label {
+  display: grid;
+  align-items: baseline;
```

Comment:
I think this `baseline` is making the labels look lower than the checkboxes

| baseline | no baseline |
|--------|-------|
|<img width="193" alt="image" src="https://github.com/user-attachments/assets/5fc1e634-41ed-4afd-95ba-ad5f9ddcf374">|<img width="232" alt="image" src="https://github.com/user-attachments/assets/8b3a81c9-70f6-4eef-99ab-154802095671">|